client/src/components/Cover/Cover.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import PendingCov from "./PendingCov";
+import "./Cover.css";
+
+export default function Cover() {
+    return (
+        <div className="all">
+            <div className="topWordssss">
+                <h1 className="tspecial">Pending Coverage</h1>
+                <h1 className="tsame">Pending Supervisor Approval</h1>
+                <h1 className="tsame">Request History</h1>
+            </div>
+            <div className="middleWordssss">
+                <h2 className="msame1">Time and Location</h2>
+                <h2 className="msame">Needing Coverage</h2>
+                <h2 className="msame">Notes</h2>
+                <h2 className="msame2"></h2>
+            </div>
+                <PendingCov
```

Comment:
This is great use of props. In order for this to be dynamic, we're gonna need to get the data from the state rather than hardcoding it like you did. Can you change this to pass the props from the state, and then just hardcode the state from now (we'll get the state from the backend soon).